### PR TITLE
Updated lnd instructions to run via tor

### PR DESCRIPTION
**Pull request by plantOS**
With latest LND commit only tor.socks, tor.dns and tor.streamisolation flags available

### CODE REVIEW

File: docs/configuring_tor.md [hunk 1]
```diff
@@ -77,47 +66,23 @@ At this point, we can now start `lnd` with the relevant arguments:
...
-service. A path to save the onion service's private key can be specified with
-the `--tor.privatekeypath` flag. A v3 onion service can also be used, but it
-must be created manually. We'll expand on how this works in [Listening for
-Inbound Connections](#listening-for-inbound-connections).
-
-Most of these arguments have defaults, so as long as they apply to you, routing
-all outbound and inbound connections through Tor can simply be done with:
 ```shell
-⛰  ./lnd --tor.active --tor.v2
+⛰  ./bitcoind --torcontrol=127.0.01:9051
```
Comment (qubenix) [inline docs/configuring_tor.md]: You missed a `.` here. Should be:

```
./bitcoind --torcontrol=127.0.0.1:9051
```

**Comment by Roasbeef** [comment]
What are you trying to "update" here? You've removed most of the instructions. I don't think you understood the prior set of instructions.

**Comment by plantOS** [comment]
@Roasbeef It's hard to understand flags that don't exist anymore. Boss, current guide is dated. 

**Current**
Currently we only have the following flags for Tor listed below:
```
--tor.socks
--tor.dns
--tor.streamisolation
```
**Old**
The following flags does not exist which rules the current guide dated listed below:

```
--tor.active
--tor.v2                                                
--tor.v3       
```                                         
Apart from that I've noticed the current guide doesn't include 
`bitcoind --torcontrol=127.0.01:9051`
which is required to get lnd routed via Tor based on my recent successful experience.
Hope this is clear enough for you now. I've messaged you on Slack as well for further clarification.

**Comment by Roasbeef** [comment]
The docs are for master. You're the one that's running a dated version of
lnd.

On Jul 8, 2018 4:17 AM, "plantOS" <notifications@github.com> wrote:

@Roasbeef <https://github.com/Roasbeef> It's hard to understand flags that
don't exist anymore. Boss, current guide is dated.

*Current*
Currently we only have the following flags for Tor listed below:

--tor.socks
--tor.dns
--tor.streamisolation

*Old*
The following flags does not exist which rules the current guide dated
listed below:

--tor.active
--tor.v2
--tor.v3

Apart from that I've noticed the current guide doesn't include
bitcoind --torcontrol=127.0.01:9051
which is required to get lnd routed via Tor based on my recent successful
experience.
Hope this is clear enough for you now. I've messaged you on Slack as well
for further clarification.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/lightningnetwork/lnd/pull/1519#issuecomment-403280494>,
or mute the thread

**Comment by plantOS** [comment]
@Roasbeef Listed below is the version of my lnd now. Is that dated?

```
commit 6382215346968b69b8b0f5553b7c0bf39e01c550
Merge: 5c42a88 4a200d2
Author: Olaoluwa Osuntokun <laolu32@gmail.com>
Date:   Wed May 23 15:39:30 2018 -0700
```

**Comment by plantOS** [comment]
@Roasbeef Thanks! Apologize for the confusion. Yes this technique only applies to that commit version. Have updated code to the following and works as described in guide on Master currently. Will close this.

```
commit 03650bbb18c6fa58bb9adc41054b20f73628bf5a
Merge: 6aaf692 92cbfbc
Author: Olaoluwa Osuntokun <laolu32@gmail.com>
Date:   Fri Jul 6 21:39:04 2018 -0500
```

**Comment by Roasbeef** [comment]
NP! Since 0.4.2 our Tor feature set has expanded significantly, so if you want to run using Tor, I would recommend upgrading to master.

**Comment by 2tbwXj46BDbdNBRV79DS** [comment]
im having this issure with 
`lnd version 0.4.2-beta commit=7cf5ebe2650b6798182e10be198c7ffc1f1d6e19`
-h Tor displays 

```
Tor:
      --tor.socks=                                            The port that
                                                              Tor's exposed
                                                              SOCKS5 proxy is
                                                              listening on.
                                                              Using Tor allows
                                                              outbound-only
                                                              connections
                                                              (listening will
                                                              be disabled) --
                                                              NOTE port must be
                                                              between 1024 and
                                                              65535
      --tor.dns=                                              The DNS server as
                                                              IP:PORT that Tor
                                                              will use for SRV
                                                              queries - NOTE
                                                              must have TCP
                                                              resolution enabled
      --tor.streamisolation                                   Enable Tor stream
                                                              isolation by
                                                              randomizing user
                                                              credentials for
                                                              each connection.


```

**Pull request closed without merging**